catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -53,6 +54,8 @@
 
   private static final String APPLICATION_JSON = "application/json";
 
+  private final MgrsCoordinateProcessor mgrsCoordinateProcessor = new MgrsCoordinateProcessor();
```

Comment:
Is there a reason why we don't instantiate this in blueprint?

Reply:
Yeah it was simpler to do it this way since we have no need to mock the processor ... yet.